homework.md

```diff
@@ -44,43 +43,43 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language for the web
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Git is a version control . It keeps track of projects , and allows many collaborors to work on a project at the same time . 
```

Comment:
Yes, it allows many **collaborators** to work on a project.

Reply:
Its great for collaborating